tensorboard/plugins/text/summary.py

```diff
@@ -79,8 +79,8 @@ def pb(name, data, display_name=None, description=None):
   Arguments:
     name: A name for the generated node. Will also serve as a series name in
       TensorBoard.
-    data: A python bytes string (str). Or a numpy array containing string data
-      (of type numpy.string_).
+    data: A python bytes string (str), a unicode string, or a numpy array
```

Comment:
Don't say "bytes string (`str`)"; this is not true in Python 3. Say "a Python bytestring (of type `bytes`), or Unicode string, or numpy array of one of these types." or similar.

Note that `str is bytes` in Python 2, while in Python 3 `bytes` is a separate type.

Reply:
Ah, indeed, thanks!